chapters/unitexpressions.tex

```diff
@@ -89,3 +89,171 @@ \section{The Syntax of Unit Expressions}\label{the-syntax-of-unit-expressions}
 
 The unit expression \lstinline!"T"! means tesla, but note that the letter \lstinline!T! is also the symbol for the prefix tera which has a multiplier value of 10\textsuperscript{12}.
 \end{example}
+
+
+\section{Unit Checking}\label{unit-checking}
+
+How to verify that units are used in compatible ways is current not fully determined by the specification.
+This section gives rules for certain situations, but in general tools are expected to reason according to standard dimensional analysis.
+
+
+\subsection{Standard Dimensional Analysis}\label{standard-dimensional-analysis}
+
+This section gives an incomplete characterization of ``standard dimensional analysis'', also referred to as just \firstuse{dimensional analysis}.
+What is described below applies to unit checking in Modelica -- \emph{dimensional analysis} could have additional meanings in other contexts.
+While Modelica has a concept of \willintroduce{empty units} (described in later sections), standard dimensional analysis only deals with non-empty units such as \lstinline!"m"!, \lstinline!"m/s"!, or \lstinline!"1"!.
+It consists of two parts:
+\begin{itemize}
+\item
+  Unit compatibility requirements.
+\item
+  Rules for deriving the unit of an expression.
+\end{itemize}
+
+Examples of unit compatibility requirements that must be met in Modelica:
```

Comment:
"Examples" seems weird here -- examples are typically for demonstrating an introduced concept, but these read like specified requirements. I assume you meant to express that this list is not complete.
It would be better if this (and other equivalent) lists were exhaustive and not qualified by descriptions like "certain somethings", "examples of",...
For example here, what about this?

> "Unit compatibility requirements that must be met in Modelica:"
And at the end have a last item be "All other situations: not yet covered by the specification".

Then, when parsing the documentation, it's much easier to see when you step off the known process flow.

Reply:
OK now?

Reply:
Yes, thanks.

Reply:
OK.  Resolving.